### ISSUE TITLE
[Feature Request] Disabling the Proximity Sensorr

### ISSUE THREAD

**Issue by KaiserCalm** [basti564/Oculess#49]
Hello Basti564,
I don't know if it's possible but it would be nice to disable the proximity sensor. 
Thank you and amazing job on the app!

**Comment by D3SOX** [comment]
I'd appreciate that because always using the PC to do it is kinda annoying. Wish there was an option for disabling it permanently.

**Comment by raykai** [comment]
I would like this option also!

**Comment by basti564** [comment]
While this is a great idea I don't believe it is possible to do this since changing system properties isn't something you can do from apps..

**Comment by KaiserCalm** [comment]
Is it possible to do for as long as the Quest stays on? Sidequest is able to do it until restart.

**Comment by Zthrowaway** [comment]
@KaiserCalm @raykai @D3SOX This is a feature available with the current version of SideQuest. You do have to restart your headset for changes to take effect, though.

**Comment by D3SOX** [comment]
No, SideQuest only has that button on the PC-side and it only works without a reboot, not after.
The information under that button is false. I think it just runs `adb shell am broadcast -a com.oculus.vrpowermanager.prox_close` which needs to be executed from a PC and doesn't persist a reboot.

**Comment by Zthrowaway** [comment]
Ah, ok then. I just watched the tutorial but I've never used it, so I don't know that much about it. Thanks for telling me!

**Comment by smartglasseshub** [comment]
I've been figuring out how to disable the Oculus Quest 2 proximity sensor, which turns off the display when the headset is removed. Here are some methods I found:

Cover the Sensor with Tape:
- The proximity sensor is located between the lenses above the IPD level indicator.
- Cover it with opaque tape to trick the headset into thinking it's still being worn.

Disable Sensor Using SideQuest:
- Use SideQuest, a software tool, after enabling developer mode on the Quest 2.
- Connect the headset to a PC or mobile device with a USB cable.
- SideQuest offers an option to disable the proximity sensor.

Use Oculus Developer Hub:
- The Oculus Developer Hub also allows for the disabling of the proximity sensor.
- This tool is easy to install and offers a straightforward option for sensor control.

Keep the Headset on Your Forehead:
- For short breaks, simply resting the headset on your forehead keeps the sensor activated.
- Useful when you need to quickly interact with your surroundings without the headset going to sleep.

Adjust Headset Sleep Settings:
- While not disabling the sensor, extending the automatic sleep period gives more time before the headset goes to sleep after removing it.

Each of these methods offers a way to keep your Quest 2 display on when the headset is not being worn. Feel free to ask for more details or check out my [article](https://smartglasseshub.com/disable-quest-2-proximity-sensor/) for a more in depth solution.
